keps/sig-network/3836-kube-proxy-improved-ingress-connectivity-reliability/README.md

```diff
@@ -424,8 +421,12 @@ Not any different than today.
 
 ###### What are other known failure modes?
 
+No other are known.
```

Comment:
The one described in Risks/Mitigations:
```
Vendors of Kubernetes which deploy Kube-proxy and specify a livenessProbe targeting /healthz are expected to start seeing a CrashLooping Kube-proxy when the Node gets tainted with ToBeDeletedByClusterAutoscaler. This is because: if we modify /healthz to fail when this taint gets added on the Node, then the livenessProbe will fail, causing the Kubelet to restart the Pod until the Node is deleted. As far as we can tell, no vendor set livenessProbe, nor does kubeadm, so the risk is low.
```

is actually relevant and we should probably mention it here [with bullet points from the template]

Reply:
Updated